draft-ietf-lamps-kyber-certificates.md

```diff
@@ -414,42 +414,97 @@ certificates.
 
 ## Example Private Key {#example-private}
 
-The following is an example of a ML-KEM-512 private key with hex seed `0001…3f`:
+The following examples show ML-KEM private keys in different formats,
+all derived from the same seed `000102…1e1f`. For each security level,
+we show the seed-only format (using explicit tag `[0]`), the expanded
```

Comment:
> using explicit tag `[0]`

"using an implicit encoding of a context-specific `[0]` tag" is more technically correct.

Reply:
[8f499e0](https://github.com/lamps-wg/kyber-certificates/pull/100/commits/8f499e07b2face998d1aea39aa3a921de29d0880)